### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by vishesh**
Quickly added a Dockerfile to make things easier to deploy.

### CODE REVIEW

**Comment by vishesh** [comment]
Depends on https://github.com/vishesh/racketscript-playground/pull/18

**Comment by stchang** [comment]
so to deploy I can just run `docker run`?

**Comment by vishesh** [comment]
Yes. I added instructions. Just `make docker-build` and `make docker-run`. I tested with port 8080, but should be fine with 80 too with right access.

I'm a Docker noob, hopefully I got this right. If I get a change I will  explore a bit of registry system and all, so that anyone can just pull images and run stuff on cloud services with one click (I remember this Deploy on Heroku button which was pretty cool).

**Comment by vishesh** [comment]
Pushing to Docker registry was easy enough. https://hub.docker.com/repository/docker/vishesh/racketscript-playground

Updated with new instructions.

File: README.md [hunk 1]
```diff
@@ -21,7 +19,27 @@ file must be `source.rkt`.
...
+# Pull docker image
+docker pull vishesh/racketscript-playground
+
+# Run playground webserver on port 8080
+docker run -dp 8080:8080 -t vishesh/racketscript-playground
+```
+
+You can also build image yourself using `make docker-build`, followed by `make
+docker-run` to start the playground web server. By default, `make docker-run`
+binds webserver to port 80, and you may want to remap that to something else
```
Comment (stchang) [inline README.md:39]: should this say port 8080?

**Comment by stchang** [comment]
it's working well for me. I had to use `sudo` to run it though?

File: README.md (hunk 1 above)
Comment (vishesh) [reply README.md:39]: Yes. Fixed.

**Comment by vishesh** [comment]
Never had to use `sudo` for 8080 port. Maybe something to do with Docker daemon config?

**Comment by stchang** [comment]
lgtm

**Pull request merged**